### PR TITLE
several updates in the interest of publishing the newsletter

### PR DESCRIPTION
**Pull request by srabun**
Newsletter: http://newsletter4.iiif.io/news/2017/08/30/newsletter/ (I have gone through and checked all of the links - they should all be correct)

Glen announcement: http://newsletter4.iiif.io/news/2017/08/30/technical-coordinator/

Event page: http://newsletter4.iiif.io/event/

Toronto working meeting page: http://newsletter4.iiif.io/event/2017/toronto/

Toronto Showcase: http://newsletter4.iiif.io/event/2017/toronto-showcase/

DC conference: http://newsletter4.iiif.io/event/2018/washington/

Community list updated: http://newsletter4.iiif.io/community/#participating-institutions

### CODE REVIEW

File: source/event/2017/toronto-showcase.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Table of Contents
+{:.no_toc}
+
+* Goes Here
+{:toc}
+
+## IIIF Showcase - Toronto - 2017
+
+Join the International Image Interoperability Framework (IIIF) community for an evening showcase to demonstrate the power and potential of interoperable image delivery on the Web. Embraced by large and growing number of the world’s leading research and cultural heritage organizations, IIIF provides an open framework for publishing digital image-based resources, to be viewed, shared, cited, annotated, and more by any compatible image-viewing application. The program will provide an introduction to IIIF, exploring how institutions are leveraging IIIF to improve image delivery solutions, while simultaneously providing new and rich features and functionality for a world-class end user experience. As a community-driven initiative, we welcome anyone with an interest to participate in this growing movement and take advantage of the common framework.
```
Comment (jpstroop) [inline source/event/2017/toronto-showcase.md]: Embraced by *a* large and growing number -- missing "*a*"

Comment (srabun) [reply source/event/2017/toronto-showcase.md]: thank you!

File: source/event/2017/toronto.md [hunk 2]
```diff
@@ -12,7 +12,11 @@ tags: [event ]
...
+The International Image Interoperability Framework ([IIIF][home-page]) Working Meeting in Toronto provides an in-person venue for active IIIF community participants and [IIIF groups][groups] to make progress on advancing specific aspects of IIIF such as specifications, software, and community development (as opposed to the annual IIIF conference, which is more geared toward sharing knowledge). Attendance is free, but capacity is limited, so [registration][registration] is required in order to ensure that we do not exceed the capacity of the venue.
```
Comment (jpstroop) [inline source/event/2017/toronto.md]: Do we need "(as opposed to the annual IIIF conference, which is more geared toward sharing knowledge)"? If so, I'd move it out of parentheses, but that's just me.

Comment (srabun) [reply source/event/2017/toronto.md]: ok

**Comment by jpstroop** [comment]
👍

**Comment by zimeon** [comment]
:+1: from a skim through (quite a lot there!), will merge

**Pull request merged**